common/production_methods/imperia_urban.txt

```diff
@@ -14,16 +14,16 @@ pm_rickshaws = {
 
 	building_modifiers = {
 		workforce_scaled = {
-			goods_input_fabric_add = 2
-			goods_input_wood_add = 4
+			goods_input_fabric_add = 1
+			goods_input_wood_add = 2
 			# output goods
-			goods_output_transportation_add = 10
+			goods_output_transportation_add = 5
```

Comment:
I changed it slightly to reduce transportation output by 20% instead of halving since pm_no_public_transport giving 5 transport for free messes with the scaling if it were to halve ( 8 -> 12 -> 16 -> 20  instead of 10 -> 15 -> 20 -> 25)